app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,25 @@
+class MessagesController < ApplicationController
+
+  def create
+    message = Message.new(message_params)
+    message.user = current_user
+    if message.save
```

Comment:
I'm guessing the reason the messages don't show up on the page when reloading is the bid might not be getting passed into the message object before it saves.